chunk 1: Set the base paragraph indentation to 5n (upstream commit 5d2e49f8).
chunk 2: Always set .Nm in bold, including in NAME (local regression fix).

Index: tmac/doc.tmac
--- tmac/doc.tmac.orig
+++ tmac/doc.tmac
@@ -178,14 +178,8 @@
 .\" via andoc.tmac and man(1); see \n[doc-hyphen-flags] in doc-common.
 .if !r HY .nr HY 1
 .
-.\" Use -rIN=<xxx> to set the paragraph indentation amount.
-.if !r IN \{\
-.  \" We select an integer indentation value in nroff mode because this
-.  \" value is used additively for multiple purposes; rounding of
-.  \" accumulating fractions would produce inconsistent results.
-.  ie t .nr IN 7.2n
-.  el   .nr IN 7n
-.\}
+.\" Use -rBP=<xxx> to set the paragraph indentation amount.
+.if !r BP .nr BP 5n
 .
 .\" LL and LT registers are handled by the doc-setup-page-layout macro.
 .
@@ -1163,13 +1157,8 @@
 .  if !\n[doc-arg-count] \{\
 .    ds doc-macro-name Nm
 .    ie \n[.$] \{\
-.      \" Handle '.Nm ...' in "Name" section: don't use a special font.
-.      ie \n[doc-in-name-section] \{\
 .        if "\*[doc-topic-name]"" \
 .          ds doc-topic-name "\$1\"
-.        No \$@
-.      \}
-.      el \
 .        doc-parse-args \$@
 .    \}
 .    el \{\
